python/cuml/manifold/t_sne.pyx

```diff
@@ -410,15 +413,15 @@ class TSNE(Base):
 
         # Clean up memory
         del _X
-        self.Y = Y
+        self.embedding_ = Y
         return self
 
     def __del__(self):
-        if "Y" in self.__dict__:
-            del self.Y
-            self.Y = None
+        if "embedding_" in self.__dict__:
+            del self.embedding_
```

Comment:
None of this should be necessary in del, by the way. But I believe it was a fix for an a lurking memory leak, so a deeper dive would probably come as a separate change.